### PR TITLE
Add commissioned word count, wordcount and path to capidata and metrics

### PR DESCRIPTION
**Pull request by Reettaphant**

### CODE REVIEW

File: src/main/scala/com/gu/editorialproductionmetricsmodels/models/EventTypes.scala [hunk 1]
```diff
@@ -12,7 +12,10 @@ case class CapiData(
    firstPublicationDate: String,
    commissioningDesk: String,
    originatingSystem: OriginatingSystem,
-   productionOffice: Option[ProductionOffice])
+   productionOffice: Option[ProductionOffice],
+   commissionedWordCount: Option[Int],
+   path: String,
+   wordCount: Option[Int])
```
Comment (susiecoleman) [inline src/main/scala/com/gu/editorialproductionmetricsmodels/models/EventTypes.scala:18]: Is there a scenario where word count isn't in CAPI?

Comment (RichieAHB) [inline src/main/scala/com/gu/editorialproductionmetricsmodels/models/EventTypes.scala:18]: Why is `wordCount` an `Option`? Would it not always have one or does it only get written on first save in composer?

**Review by susiecoleman — APPROVED** [review]
This looks fine to merge. However if it is the case that either `commissionedWordCount` or `wordCount`are always in CAPI I would make them non optional. It's just a standard sonotype release let me know if you have any questions. It can take a couple of hours to show up so we should release it as soon as we can.

File: src/main/scala/com/gu/editorialproductionmetricsmodels/models/EventTypes.scala (hunk 1 above)
Comment (susiecoleman) [reply src/main/scala/com/gu/editorialproductionmetricsmodels/models/EventTypes.scala:18]: Have discussed this with Reetta. The word count is an option in our database as it is a field that has been added retrospectively so not all entries have it. It is also optional from in the capi scala client so I think leaving it as optional makes sense.

**Pull request merged**